Integrations/integration-AnomaliThreatStream.yml

```diff
@@ -0,0 +1,433 @@
+description: Use Anomali ThreatStream to query and submit threats
+configuration:
+- display: Server URL (e.g. https://192.168.0.1)
+  name: server
+  defaultvalue: https://api.threatstream.com
+  type: 0
+  required: true
+- display: Username
+  name: username
+  defaultvalue: ""
+  type: 0
+  required: false
+- display: API Key
+  name: apikey
+  defaultvalue: ""
+  type: 4
+  required: false
+- display: Do not validate server certificate (insecure)
+  name: insecure
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Use system proxy settings
+  name: proxy
+  defaultvalue: ""
+  type: 8
+  required: false
+script:
+  script: |+
+    var base = params.server.replace(/[\/]+$/, '') + '/api/';
+    args.username = params.username;
+    args.api_key = params.apikey;
+
+    var commandDict = {
+        'test-module': {
+            method: 'GET',
+            url: 'intelligence/',
+            version: 'v2',
+        },
+        'threatstream-intelligence': {
+            method: 'GET',
+            url: 'intelligence/',
+            version: 'v2',
+        },
+    }
+
+    var sendRequest = function(method, url, version, args, headers, multipart) {
+        url = base + version + '/' + url;
+        if (method === 'GET') {
```

Comment:
At the moment we don't (I also removed the "import" functions which you implemented), but we may do so in the future. @NeelRust asked for the "upload file" feature (currently not implemented) which will require the "post" and "multipart" features.
I can remove it, but it works now and will probably be used in the future. Whatever you think.